addons/account_edi/models/account_edi_format.py

```diff
@@ -387,6 +430,8 @@ def _create_invoice_from_attachment(self, attachment):
                     elif file_data['type'] == 'pdf':
                         res = edi_format._create_invoice_from_pdf_reader(file_data['filename'], file_data['pdf_reader'])
                         file_data['pdf_reader'].stream.close()
+                    else:  # file_data['type'] == 'binary'
```

Comment:
If you're going to put the condition as a comment why not just... actually put the condition in? The previous versions did not bother falling through, the new one probably should not either.

Maybe a handler triggering an error on unknown types could be a good idea, but assuming anything which is not an XML or a PDF is a binary seems less than ideal?

Reply:
It's an added feature that assuming it's not an XML or PDF (or anything else that would be latter supported) we fall back to 'binary'. I agree however that the comment is maybe superfluous.